bin/rtlcss.js

```diff
@@ -197,8 +205,9 @@ if (!shouldBreak) {
               }
             })
           } else {
-            // process only *.css
-            if (/\.(css)$/.test(file)) {
+            // process only *.css and ignore RTLed files
+            var patt = new RegExp('('+ext+')', 'gm');
```

Comment:
You'll need more than that! You have to [escape](https://github.com/MohammadYounes/rtlcss/blob/master/lib/util.js#L36-L38) the whole string